app/views/splash/index.html.erb

```diff
@@ -1,6 +1,12 @@
 <div class="splash-container">
    <h1 class="splash-title">BUDGETARY APP TRANSACTIONS</h1>
+   <div class ="sign-in">
+  <%= link_to "Sign in", new_user_session_path, method: :delete, class:"nav-link1" %>
+
+  <%= link_to "Sign up", new_user_registration_path, method: :delete, class:"nav-link2" %>
```

Comment:
- [ ] Sign up a new user throws an error, when the signup form is submitted. As mentioned in the previous reviews kindly ensure to improve upon the same.

**Sign up error:**
![image](https://user-images.githubusercontent.com/12745474/204113852-40d8e332-5b9e-41aa-9411-9741b54af73f.png)

Reply:
Done